CMakeLists.txt

```diff
@@ -644,9 +640,36 @@ endif()
 
 set(CPACK_PACKAGE_DIRECTORY "dist")
 set(CPACK_PACKAGE_FILE_NAME "isle-${PROJECT_VERSION}-${ISLE_PACKAGE_NAME}-${CMAKE_SYSTEM_PROCESSOR}")
+
+find_program(DPKG dpkg)
 if(MSVC)
   set(CPACK_GENERATOR ZIP)
+elseif(DPKG)
+  set(CPACK_DEBIAN_PACKAGE_DESCRIPTION "LEGO Island for Debian-based distros")
+  set(CPACK_DEBIAN_PACKAGE_VERSION ${PROJECT_VERSION})
+  set(CPACK_DEBIAN_PACKAGE_DEPENDS "libqt6core6t64 libsdl3-0")
+  set(CPACK_DEBIAN_PACKAGE_SECTION games)
+  set(CPACK_DEBIAN_PACKAGE_PRIORITY optional)
+  set(CPACK_DEBIAN_PACKAGE_HOMEPAGE "https://github.com/isledecomp/isle-portable")
+  set(CPACK_DEBIAN_PACKAGE_MAINTAINER "Joshua Peisach <itzswirlz2020@outlook.com>")
+
+  # Don't replace system SDL3
+  list(REMOVE_ITEM install_extra_targets "SDL3-shared")
```

Comment:
It's build on Ubuntu 24.04 which doesn't even have a package for SDL3 yet, that feels a bit problematic.

Reply:
True.. that would move us to 25.04 or later